src/content/configuration/other-options.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Other Options
-sort: 19
+sort: 20
```

Comment:
Why this order?

Reply:
Other options at the bottom as they dont have dedicated section (and they were at the bottom).